code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -82,6 +83,7 @@
 /mob/living/simple_animal/drone/polymorphed/Initialize()
 	. = ..()
 	liberate()
+	pacifist = FALSE
```

Comment:
Move this out of initialize and into the typepath, else it just won't work and leave poly drones as pacifists.